OneCommunityGlobal/HGNRest
# Nathalia new filter project by user name

# Description
![image](https://github.com/OneCommunityGlobal/HGNRest/assets/69649453/7cb51812-3900-4be0-ac1f-3b99ab0b4aa7)

## Related PRS (if any):
This backend PR is related to the #[2392](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2392) frontend PR.

## Main changes explained:
- Created the getProjectByPerson function in userProfileController.js
- Created the route to call the getProjectByPerson function

## How to test:
1. Check into current branch
2. Run `npm install`, `npm run build` and `npm run start` to run this PR locally
3. Go to the #[2392](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2392) frontend PR and follow the steps

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HGNRest/assets/69649453/76c3724e-b253-495a-83e5-beb713d17c61



## Note:
It's really important to run `npm run build`
